dashboard/src/actions/toastActions.js

```diff
@@ -25,3 +25,11 @@ export const showFailureToast = () => async (dispatch) => {
     payload: { ...toast },
   });
 }
+
+export const constructToast = (variant, title, message ="") => async(dispatch) => {
+  let toast = { variant, title, message };
+  dispatch({
+    type: TYPES.SHOW_TOAST,
+    payload: { ...toast },
+  });
```

Comment:
Why bother creating `toast` and then unpacking it again?...why not just use the values in-line:
```
  dispatch({
    type: TYPES.SHOW_TOAST,
    payload: { variant, title, message },
  });
```